src/commands/deploy.ts

```diff
@@ -72,9 +77,15 @@ export async function deployCommand(options: GenezioDeployOptions) {
     // Otherwise, the user will get an error at runtime. This check can be removed in the future once no one is using version
     // 0.1.* of @genezio/types.
     const packageJsonPath = path.join(backendCwd, "package.json");
-    if (isDependencyVersionCompatible(packageJsonPath, "@genezio/types", REQUIRED_GENEZIO_TYPES_VERSION_RANGE) === false) {
+    if (
+        isDependencyVersionCompatible(
+            packageJsonPath,
+            "@genezio/types",
+            REQUIRED_GENEZIO_TYPES_VERSION_RANGE,
+        ) === false
+    ) {
```

Comment:
I know you did not add this, but this is only valid for ts/js projects. It actually writes an error in the terminal when deploying.

```
ERROR: Error while reading package.json file: Error: ENOENT: no such file or directory, open '/home/costin/genezio/my-projects/gotest/package.json'
```

Maybe we should extract this logic into a function and treat it based on the project language. This function is also called in `genezio local`, so it should also be modified there.

Reply:
The error is loggerDebug it won't appear in production.

Reply:
Even though is not shown in production, I still think we should only check this for ts/js environments.